solr_scripts/schema.xml

```diff
@@ -215,6 +215,4 @@
 
   <!-- SolrQueryParser configuration: defaultOperator="AND|OR" -->
   <solrQueryParser defaultOperator="AND"/>
-  <similarity class="solr.BM25SimilarityFactory"/>
```

Comment:
Lets add the BM25 similarity scorer back in. The default for this version of solr is TFIDF which is not as good.